django_tables2/columns/base.py

```diff
@@ -197,12 +197,15 @@ class Column(object):
               - If `True`, force localization
               - If `False`, values are not localized
               - If `None` (default), localization depends on the ``USE_L10N`` setting.
-        linkify (bool, str, callable, dict): Controls if cell content will be wrapped in an
-            ``a`` tag. If True, the model's ``get_absolute_url`` is used as the
-            ``href`` attribute. If a callable is passed, the returned value is used,
-            if it's not ``None``.
-            If a dict is passed, it's passed as keyword arguments to
-            ``~django.urls.reverse``.
+        linkify (bool, str, callable, dict, tuple): Controls if cell content will be wrapped in an
+            ``a`` tag. The different ways to define the ``href`` attribute:
+
+             - If `True`, the ``record.get_absolute_url()`` or the related model's
+               `get_absolute_url()` is used.
```

Comment:
What happens if this does not exist or returns a non-string?

Reply:
Code says: TypeError if it doesn't exist, no check for result I think. Might be worth giving it another glance and document a little more. Seems likely ppl will run into this when they just set link to True. Unlikely that `get_absolute_url` returns non-string. When it returns `None`, what happens?

Reply:
If `get_absolute_url` does not exist -> error message: `If linkify=True '{}' must have a method get_absolute_url`.

The return value is used verbatim in the template, so I think that should be transparent enough?